recipe/meta.yaml

```diff
@@ -9,20 +9,18 @@ source:
   sha256: 0ca7568c8be919b69f99644d29d31738a5b1a87750d06dd36564bcfad278d402
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
-  skip: true  # [win]
+  skip: true  # [py>=313]
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ stdlib("c") }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
Why were these lines removed?

Reply:
I guess I don't understand it either - is this package all of a sudden noarch?  Or has it always been noarch and it was only realised now?